### DISCUSSION TITLE
Where can I find https://github.com/Pix4D/concourse-pool-boy?

### DISCUSSION THREAD

**Discussion by lxiaopei** [concourse/concourse#6062] [General]
Anyone knows where I can find https://github.com/Pix4D/concourse-pool-boy?
I am using concourse pool resource https://github.com/concourse/pool-resource
and sometimes when ensure fails, the lock becomes stale. 

I googled and found https://github.com/Pix4D/concourse-pool-boy is to fix this, but the link is broken.  Thanks in advance for your help~

- name: acquire-1
  plan:
    - put: acquire-pool
      params: {acquire: true}
    - task: BLAHBLAHBLAH
  ensure:
    put: acquire-pool
    params: {release: acquire-pool}

**Comment by taylorsilva** [comment]
You could try emailing the org @Pix4D to see if they have a copy still. I checked to see if it was in the wayback machine (http://web.archive.org/) but no luck :(

**Comment by marco-m-pix4d** [comment]
As answered on https://github.com/concourse/pool-resource/issues/39, the Pool Boy is back. https://github.com/Pix4D/concourse-pool-boy
